### PR TITLE
Allow setting formatting options in vimrc

### LINKED ISSUE #4: Add options for JuliaFormatter.jl
Currently all options are hardcoded:

https://github.com/kdheepak/JuliaFormatter.vim/blob/4a56758c60f20d1858c686596b35f6730cb8fefb/scripts/server.jl#L36-L40

### PR DESCRIPTION
**Pull request by jkozdon**
closes #4

### CODE REVIEW

**Comment by jkozdon** [comment]
I'm not a vim plugin guru, but this seems to work in vim (not sure about neovim)

File: autoload/JuliaFormatter.vim [hunk 1]
```diff
@@ -2,6 +2,10 @@ if get(g:, 'JuliaFormatter_loaded')
     finish
 endif
 
+if !exists("g:JuliaFormatter_options")
+  let g:JuliaFormatter_options = ''
+endif
+
```
Comment (kdheepak) [inline autoload/JuliaFormatter.vim:8]: I think these lines should be in `plugin/JuliaFormatter.vim`.

**Comment by kdheepak** [comment]
Thanks for making the PR! I've been meaning to get around to this.

**Comment by kdheepak** [comment]
It would be nice if all the options were separate. I can merge this and make changes locally if you don't mind.

**Comment by jkozdon** [comment]
I just pushed out a cleaned up version.

> It would be nice if all the options were separate. I can merge this and make changes locally if you don't mind.

Feel free to push to my branch or merge and clean up. Your call.

File: autoload/JuliaFormatter.vim (hunk 1 above)
Comment (jkozdon) [reply autoload/JuliaFormatter.vim]: Fixed

**Comment by jkozdon** [comment]
> > It would be nice if all the options were separate. I can merge this and make changes locally if you don't mind.

Are you thinking one option for each of the supported Julia Formatter options? If so, can you bump the version first like in #7?

I need the updated JuliaFormatter for the [project](https://github.com/climate-machine/CLIMA/blob/6ec24260d1669b9ad3dcae9f6b6dc41d02b096ca/.dev/clima_formatter_options.jl) I work on.

**Comment by kdheepak** [comment]
> I just pushed out a cleaned up version.

Nice! 

> Feel free to push to my branch or merge and clean up. Your call.

I'll checkout the branch locally, cherry pick your commit and work on that to make this change.

> Are you thinking one option for each of the supported Julia Formatter options?

Yes. It seems like that would be idiomatic vim.

**Comment by jkozdon** [comment]
Sounds good.

**Comment by kdheepak** [comment]
Changes merged into master in this commit: https://github.com/kdheepak/JuliaFormatter.vim/commit/014af3cc0ebe86223f40d1f5780897e9ab7389de

See https://github.com/kdheepak/JuliaFormatter.vim/commit/7d215acfa5db62372e16acffc46ac64047561ef0 for implementation discussed here.

**Pull request closed without merging**